### PR TITLE
Fix two bugs + cards from website

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/sunpy-sphinx-theme/issues/256
Fixes https://github.com/sunpy/sunpy-sphinx-theme/issues/257

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -67,6 +67,8 @@
     "reproject": ("https://reproject.readthedocs.io/en/stable/", None),
 }
 html_theme = "sunpy"
+html_static_path = ["_static"]
+html_extra_path = ["_static/img"]
```
Comment (nabobalis) [inline docs/conf.py:71]: Ive not set this up correctly.

**Comment by Cadair** [comment]
The lack of hover-over on the card, and the fact that only the "more info" button is clickable rather than the whole card isn't ideal.

Also this has apparently broken sunpy.org?

**Comment by Cadair** [comment]
Yeah, I just rebuilt sunpy.org and the build succeeded but the cards are broken.

**Comment by Cadair** [comment]
It is also broken on the preview build on this PR https://output.circle-artifacts.com/output/job/5f17f6e9-6b09-4173-84d9-dccb1a3a1365/artifacts/0/.tmp/website/sunpy.org/_build/html/about/project/index.html

**Comment by nabobalis** [comment]
> The lack of hover-over on the card, and the fact that only the "more info" button is clickable rather than the whole card isn't ideal.

This has always been the case tho?

> It is also broken on the preview build on this PR [output.circle-artifacts.com/output/job/5f17f6e9-6b09-4173-84d9-dccb1a3a1365/artifacts/0/.tmp/website/sunpy.org/_build/html/about/project/index.html](https://output.circle-artifacts.com/output/job/5f17f6e9-6b09-4173-84d9-dccb1a3a1365/artifacts/0/.tmp/website/sunpy.org/_build/html/about/project/index.html)

Hmm, I think I need to pass the path down to the card for the images.

**Pull request merged**